docs(install) add DB-Less / declarative config install steps (#1281)
Kong/docs.konghq.com: app/install/macos.md

    $ brew tap kong/kong
    $ brew install kong
    ```

> 2. **(DB-less only) Write declarative configuration file**

Reviewer: This could be merged with the section below as `Prepare your configuration`, with subsections for "Without a database", "Postgres" and "Cassandra".

    Follow the [Declarative Configuration Format]: /{{site.data.kong_latest.release}}/db-less-and-declarative-config/#the-declarative-configuration-format instructions while doing so.

    We'll assume the file is named `kong.yml`.

1. **Add `kong.conf`**

    **Note**: This step is **required** if you are using Cassandra; it is **optional** for Postgres users.

    By default, Kong is configured to communicate with a local Postgres instance.
    If you are using Cassandra, or want to run Kong in DB-less mode, or need to modify any settings, download the [`kong.conf.default`](https://raw.githubusercontent.com/Kong/kong/master/kong.conf.default) file and [adjust][configuration] it as necessary.
    Then, as root, add it to `/etc`:

    ```bash
    $ sudo mkdir -p /etc/kong
    $ sudo cp kong.conf.default /etc/kong/kong.conf
    ```

    In DB-less mode you will need to set the `database` config option to `off` and the `declarative_config` one to the
    absolute path of `kong.yml`:

    ```conf
    database = off
    ...
    declarative_config = /path/to/kong.yml

    ```

> 1. **Prepare your database or declarative config file**

Reviewer: `1.` coming after `2.`

    $ brew install kong
    ```

1. **Prepare your database or declarative configuration file**

    Kong can run either with or without a database.

    When using a database, you will use the `kong.conf` configuration file for setting Kong's
    configuration properties at start-up and the database as storage of all configured entities,

>     such as the routes and services to which Kong proxies.

Reviewer 2: ```suggestion
    such as the Routes and Services to which Kong proxies.
```

>     When not using a database, you will use `kong.conf` its configuration properties and a `kong.yml`

Reviewer 2: ```suggestion
    When not using a database, you will use `kong.conf`'s configuration properties and a `kong.yml`
```

    ```bash
    $ kong migrations bootstrap [-c /path/to/kong.conf]
    ```
    By default, Kong is configured to communicate with a local Postgres instance.

>     If you are using Cassandra, or need to modify any settings, download the [`kong.conf.default`](https://raw.githubusercontent.com/Kong/kong/master/kong.conf.default) file and [adjust][configuration] it as necessary.

Reviewer 2: ```suggestion
    If you are using Cassandra, or need to modify any settings, download the [`kong.conf.default`](https://raw.githubusercontent.com/Kong/kong/master/kong.conf.default) file and [adjust][configuration] as necessary.
```

>     Then, as root, add it to `/etc`:

Reviewer 2: ```suggestion
    Then, as root, add `kong.conf.default` to `/etc`:
```

    ```bash
    $ sudo mkdir -p /etc/kong
    $ sudo cp kong.conf.default /etc/kong/kong.conf
    ```

    **Note for Kong < 0.15**: with Kong versions below 0.15 (up to 0.14), use
    the `up` sub-command instead of `bootstrap`. Also note that with Kong <
    0.15, migrations should never be run concurrently; only one Kong node
    should be performing migrations at a time. This limitation is lifted for
    Kong 0.15, 1.0, and above.

    **Without a database**

    If you are going to run Kong in [DB-less mode](/{{site.data.kong_latest.release}}/db-less-and-declarative-config/),

>     you should start by generating declarative config file. The following command will generate a `kong.yml`

Reviewer 2: ```suggestion
    you should start by generating a declarative config file. The following command will generate a `kong.yml`
```

>     file in your current folder. It contains instructions about how to fill it up.

Reviewer 2: ```suggestion
    file in your current folder. It contains instructions about how to populate it.
```

    ``` bash
    $ kong config init
    ```

>     After filling up the `kong.yml` file, edit your `kong.conf` file. Set the `database` option

Reviewer 2: ```suggestion
    After populating the `kong.yml` file, edit your `kong.conf` file. Set the `database` option
```
